dv-export/python/query-changes/README.md

```diff
@@ -1,49 +1,34 @@
-# DV Query History
+# Data Version Query History
 
-DV Query History is a sample application that demonstrates the use of Visier APIs to fetch and analyze changes in your
-Visier tenant.
+The Data Version (DV) Query History sample application uses Visier APIs to fetch and analyze changes in your Visier tenant.
 
 ## Overview
 
+Let’s say we want to maintain a database that restates all changes every time an analytic object changes in Visier. We can use the Data Version Export APIs to retrieve the changes between two data versions in a Visier tenant, which allows us to identify objects that changed and the changes that occurred. However, the DV Export API doesn't retrieve changes to calculated fields, like calculated properties. To get the full change history of all attributes including calculated fields, we can use the Data Query APIs to retrieve specific attributes with their full history that weren't retrieved by the DV Export API. After we have all attributes using both APIs, we can then create a table that saves all received data in a database. 
+
 The application performs the following operations:
 
-1. **Load Data Version Changes**: The application runs an export job to get changes between two data versions inside
-   the `DVManager` class.
-   It uses the `DVExportApiClient` to schedule a data version export job and waits for it to complete.
-   Once the job is complete, it downloads file with changes for a specific analytic object.
+1. **Load data version changes**: The application uses `DVExportApiClient` to run an export job to retrieve the delta (differences) between two data versions in the `DVManager` class. The application waits for the job to complete. After the job completes successfully, the application downloads a file containing the changes to a specific analytic object (new columns, deleted columns, updated columns).
 
-2. **Get Analytic Object Filter Property**: From the downloaded file, the application selects the filter property from
-   the query template and reduces repeated values.
+2. **Select analytic object attributes**: From the downloaded export file, the application selects attributes to include in a list query. These are the attributes that the DV Export API didn't retrieve history for. The application also reduces repeated values.
```

Comment:
Normally it could be both types of attributes - canonical and calculated. Query will get all attributes which is included in queries/{query}.json.